view/frontend/templates/product/label.phtml

```diff
@@ -13,16 +13,58 @@
  */
 
 /**
- * Product discount visual template
+ * Product label visual template
  *
  * @var $block \Smile\ProductLabel\Block\ProductLabel\ProductLabel
  */
 ?>
-<?php $productLabels = $block->getProductLabels(); ?>
-<div class="productlabel-product-view">
-    <?php foreach($productLabels as $productLabel) :?>
-        <div class="<?php echo $productLabel['position_product_view']; ?>">
-            <img src="<?php echo $block->getImageUrl($productLabel['image']); ?>"/>
-        </div>
-    <?php endforeach; ?>
-</div>
+<?php if($block->getProductLabels()) : ?>
+    <?php $productLabels = $block->getProductLabels(); ?>
+    <div class="productlabel-wrapper" id="labels">
```

Comment:
You cannot have several elements with the same `id` on an HTML page. 

Also, "labels" is probably not a proper way to name your element, because it might conflict with other developers having the same idea as you. Probably something like `smile-productlabels-wrapper` is a less conflicting name.

Reply:
Done